app/src/main/java/com/framgia/fsalon/screen/report/ReportViewModel.java

```diff
@@ -0,0 +1,145 @@
+package com.framgia.fsalon.screen.report;
+
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import android.graphics.drawable.Drawable;
+import android.support.annotation.IntDef;
+import android.support.v4.app.Fragment;
+import android.view.View;
+
+import com.framgia.fsalon.BR;
+import com.framgia.fsalon.R;
+import com.framgia.fsalon.screen.homeadmin.HomePagerAdapter;
+import com.framgia.fsalon.screen.report.reportsales.ReportSalesFragment;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.framgia.fsalon.screen.report.ReportViewModel.TabButton.TAB_BILLS;
+import static com.framgia.fsalon.screen.report.ReportViewModel.TabButton.TAB_CUSTOMERS;
+import static com.framgia.fsalon.screen.report.ReportViewModel.TabButton.TAB_SALES;
+
+/**
+ * Exposes the data to be used in the Report screen.
+ */
+public class ReportViewModel extends BaseObservable implements ReportContract.ViewModel {
+    private ReportContract.Presenter mPresenter;
+    private HomePagerAdapter mAdapter;
+    private int mTabPosition;
+    private Drawable mIsSale;
+    private Drawable mIsCustomer;
+    private Drawable mIsBill;
+    private Fragment mFragment;
+
+    public ReportViewModel(Fragment fragment) {
+        mFragment = fragment;
+        setIsBill(mFragment.getResources().getDrawable(R.drawable.bg_report));
+        setIsCustomer(mFragment.getResources().getDrawable(R.drawable.bg_report));
+        setIsSale(mFragment.getResources().getDrawable(R.drawable.bg_report_selected));
+        List<Fragment> fragments = new ArrayList<>();
+        fragments.add(ReportSalesFragment.newInstance());
+        fragments.add(new Fragment());
+        fragments.add(new Fragment());
+        mAdapter = new HomePagerAdapter(fragment.getChildFragmentManager(), fragments);
+        setAdapter(mAdapter);
+    }
+
+    @Bindable
+    public Drawable getIsSale() {
+        return mIsSale;
+    }
+
+    public void setIsSale(Drawable isSale) {
+        mIsSale = isSale;
+        notifyPropertyChanged(BR.isSale);
+    }
+
+    @Bindable
+    public Drawable getIsCustomer() {
+        return mIsCustomer;
+    }
+
+    public void setIsCustomer(Drawable isCustomer) {
+        mIsCustomer = isCustomer;
+        notifyPropertyChanged(BR.isCustomer);
+    }
+
+    @Bindable
+    public Drawable getIsBill() {
+        return mIsBill;
+    }
+
+    public void setIsBill(Drawable isBill) {
+        mIsBill = isBill;
+        notifyPropertyChanged(BR.isBill);
+    }
+
+    /**
+     * TabButton
+     */
+    @IntDef({TAB_SALES, TAB_CUSTOMERS, TAB_BILLS})
+    @Retention(RetentionPolicy.SOURCE)
+    public @interface TabButton {
+        int TAB_SALES = 0;
```

Comment:
ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.